### PR TITLE
PIC-1878 Create report in database

### PR DESCRIPTION
**Pull request by theDustRoom**
I've decided to seed data as part of the database migrations, I've done this as we'll want to be able to test each report in all environments and it will reduce the number of reports generated through testing.

We also always used test events for reports in NewTech so this gives us the same (but is better/easier).

Can review this later if we decide we don't want dummy reports in the database.

I've mocked ReportService in each of the unit tests as these will change over time as we start fetching and updating data in the next few stories.

### CODE REVIEW

File: server/services/reportService.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import Report from '../repositories/entities/report'
+import ReportDefinition from '../repositories/entities/reportDefinition'
+
+export interface IReport {
+  id?: string
+  reportDefinitionId: number
+}
+
+export default class ReportService {
```
Comment (jhevans) [inline server/services/reportService.ts:11]: In all the tests I've seen this service is mocked out, are there any tests that test that data is persisted?

**Review by jhevans — COMMENTED** [review]
Code looks good, can't see test coverage for the persistence though so left one comment about that

File: server/services/reportService.ts (hunk 1 above)
Comment (theDustRoom) [reply server/services/reportService.ts:11]: Ack, had the initial UI screen creating reports but decided that'd be bad right now so removed it. Forgot to include the test elsewhere. I'll get on it.

Comment (theDustRoom) [reply server/services/reportService.ts:11]: Added a few integration tests to run through the full stack

**Review by jhevans — APPROVED** [review]

**Pull request merged**